### PR TITLE
Add a CDN link

### PR DESCRIPTION
**Pull request by mldk-tech**
#85

### CODE REVIEW

File: ibli/modules/custom/ibli_general/ibli_general.install [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+/**
+ * Enable module blog.
+ */
+function ibli_general_update_7000() {
```
Comment (amitaibu) [inline ibli/modules/custom/ibli_general/ibli_general.install]: no updates needed

Comment (mldk-tech) [inline ibli/modules/custom/ibli_general/ibli_general.install]: Known, it has been declined, but not merged. 
But I'll do it here too

**Comment by NofarGizra** [comment]
@DavidGizra 
This pull request includes changes that are not realted to the CDN link...
Please re-create the branch, by checking-out from the **master** branch.

**Comment by mldk-tech** [comment]
This is because the master not was a update, and my branch was ahead of 'origin/master.
Solved with reset the master from my computer and cherry-pick the 9ef7cd6 commit to #85a branch.

**Comment by NofarGizra** [comment]
Closed in favor of #91 .

**Pull request closed without merging**